prospector/core/prospector.py

```diff
@@ -36,7 +37,7 @@
 ONE_YEAR = 365 * SECS_PER_DAY
 
 MAX_CANDIDATES = 2000
-DEFAULT_BACKEND = "http://localhost:8000"
+DEFAULT_BACKEND = "http://backend:8000"
```

Comment:
I think it still makes sense to assume the default is localhost (no docker); in the docker configuration, this value will be overridden